.github/workflows/common-commitlint.yml

```diff
@@ -14,6 +20,7 @@ jobs:
         uses: extenda/actions/commitlint@v0
         with:
           message: ${{ github.event.pull_request.title }}
+          require-type-and-scope: ${{ inputs.require-type-and-scope }}
```

Comment:
Quote this input value to ensure it doesn't become a boolean in the YAML parser.